tests/unit/browser/webengine/test_spell.py

```diff
@@ -0,0 +1,95 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2017 Michal Siedlaczek <michal.siedlaczek@gmail.com>
+
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from os.path import basename, join
+
+import pytest
+
+from qutebrowser.browser.webengine import spell
+
+AFRIKAANS = spell.Language('af-ZA',
+                           'Afrikaans (South Africa)',
+                           'af-ZA-3-0.bdic')
+ENGLISH = spell.Language('en-US',
+                         'English (United States)',
+                         'en-US-7-1.bdic')
+POLISH = spell.Language('pl-PL',
+                        'Polish (Poland)',
+                        'pl-PL-3-0.bdic')
+
+LANGUAGE_LIST = [AFRIKAANS, ENGLISH, POLISH]
+
+
+def test_get_installed_languages_empty(tmpdir, mocker):
+    mocker.patch('qutebrowser.browser.webengine.spell.get_dictionary_dir',
```

Comment:
For tests where you need to patch something but don't need the unittest.mock `assert_*` methods, please use [monkeypatch](https://docs.pytest.org/en/latest/monkeypatch.html)